docs/release-notes/8.7.asciidoc

```diff
@@ -0,0 +1,81 @@
+[[release-notes-header-8.7.0]]
+== 8.7
+
+[discrete]
+[[release-notes-8.7.0]]
+=== 8.7.0
+
+[discrete]
+[[known-issue-8.7.0]]
+==== Known issues
+* After your instance of {elastic-sec} generates alerts for the first time, you may have to refresh your browser before your alert data will appear (navigating between pages will not suffice).
+
+[discrete]
+[[breaking-changes-8.7.0]]
+==== Breaking changes
+
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+There are no breaking changes in 8.7.0.
+// end::breaking-changes[]
+
+
+[discrete]
+[[deprecations-8.7.0]]
+==== Deprecations
+There are no deprecations in 8.7.0.
+
+
+[discrete]
+[[features-8.7.0]]
+==== Features
+* Creates a new dashboard, Data Quality, which highlights any issues in your ECS field mappings ({pull}150063[#150063]).
+* Creates a Torq connector that can trigger Torq workflows ({pull}149405[#149405]).
```

Comment:
This needs to be added to our docs (I'll file a doc issue for it) and I believe it needs to be added to the kibana release notes. I'll check with Kaarina and Lisa before I add it.

Reply:
Will keep in RN but won't mention in release highlights.